cli/running/running.go

```diff
@@ -366,7 +366,7 @@ func FillCtx(cliOpts *config.CliOpts, cmdCtx *cmdcontext.CmdCtx,
 	runningCtx *RunningCtx, args []string) error {
 	var err error
 
-	if len(args) > 1 && cmdCtx.CommandName != "run" {
+	if len(args) > 1 && cmdCtx.CommandName != "run" && cmdCtx.CommandName != "connect" {
```

Comment:
Will the error be printed on attempt to provide multiple instances for the connect command?